haikudepotserver-webapp/src/main/webapp/js/app/passivecontent/pkgscreenshotarchiveimporthelp_ja.html

```diff
@@ -0,0 +1,20 @@
+<!--
+  ~ Copyright 2017, Andrew Lindesay
```

Comment:
Can you please replace this copyright with your own name.

Reply:
Hi, I updated copyright.